### PR TITLE
Small tweaks to job refactor

### PR DESCRIPTION
**Pull request by nllong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
-Documentation resides in the [wiki](https://github.com/NREL/alfalfa/wiki)!
+# Alfalfa
+
```
Comment (TShapinsky) [inline README.md:2]: :)

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
...
+Alfalfa is an open source web application forged in the melting pot of Building Energy Modeling (BEM), Building Controls, and Software Engineering domain expertise.​ Alfalfa transforms a Building Energy Models (BEMs) into virtual buildings by providing industry standard building control interfaces for interacting with models as they run.​ From a software engineering perspective, Alfalfa leverages widely adopted open source products and is architected according to best practices for a robust, modular, and scalable architecture.
+
+## User Documentation
+
+Documentation resides in the [GitHub wiki](https://github.com/NREL/alfalfa/wiki)!
+
+## Developer Documentation
+
+We are currently working on increasing our developer documentation. See how to run the tests on the [GitHub wiki](https://github.com/NREL/alfalfa/wiki/Running-Tests). For realeasing, see the wiki's [release instructions](https://github.com/NREL/alfalfa/wiki/Release-Instructions).
```
Comment (nllong) [inline README.md:11]:
```suggestion
We are currently working on increasing our developer documentation. See how to run the tests on the [GitHub wiki](https://github.com/NREL/alfalfa/wiki/Running-Tests). For releasing, see the wiki's [release instructions](https://github.com/NREL/alfalfa/wiki/Release-Instructions).
```

File: pyproject.toml [hunk 3]
```diff
@@ -33,10 +33,13 @@ Cython = "^0.29.28"
...
 influxdb = "^5.3.1"
-matplotlib = "3.3.3"
+matplotlib = "3.5.2"
 
 [tool.poetry.dev-dependencies]
-alfalfa-client = {git = "https://git@github.com/nrel/alfalfa-client.git", rev = "40546ed"} # required for tests to pass right now
+# required for tests to pass right now. There are often issues when pinning to a branch
+# and calling poetry update. I have had to completely remove my venv and reinstall in order
+# to get the latest updates.
+alfalfa-client = {git = "https://git@github.com/nrel/alfalfa-client.git", branch = "job_refactor"}
```
Comment (nllong) [inline pyproject.toml:42]:
```suggestion
alfalfa-client = {git = "https://git@github.com/nrel/alfalfa-client.git", branch = "develop"}
```

Comment (nllong) [reply pyproject.toml:42]: pointing to develop now that it has been merged

File: README.md (hunk 1 above)
Comment (nllong) [reply README.md:2]: @anyaelena  -- can you review this README before we merge into Toby's branch?

**Review by anyaelena — APPROVED** [review]
Thanks @nllong !

**Comment by anyaelena** [comment]
Merging this and we can solve the simulation tests on pull_request issue in the Job Refactor PR.

**Pull request merged**